tests/test_scripts.py

```diff
@@ -2777,6 +2776,49 @@ def test_trim_low_abund_trimtest_savetable():
                 'GGTTGACGGGGCTCAGGGGGCGGCTGACTCCGAGAGACAGCA'
 
 
+def test_counting_load_gzipped_bigcount():
```

Comment:
This function should be moved into test_counting_hash.py, as should the above function, test_abundance_dist_nobigcount.